source/conf.py

```diff
@@ -233,3 +235,6 @@ def autodoc_process_docstring(app, what, name, obj, options, lines):
     app.connect('autodoc-process-signature', autodoc_process_signature)
     app.connect('autodoc-skip-member', autodoc_skip_member)
     app.connect('autodoc-process-docstring', autodoc_process_docstring)
+
+# link that is used to download mathjax
```

Comment:
Is this essential? On first glance, the result appeared quite similar when dropping it.